aggsender/db/migrations/0002.sql

```diff
@@ -5,7 +5,7 @@ ALTER TABLE certificate_info_history DROP COLUMN aggchain_proof;
 ALTER TABLE certificate_info_history DROP COLUMN finalized_l1_info_tree_root;
 
 -- +migrate Up
-ALTER TABLE certificate_info ADD COLUMN aggchain_proof VARCHAR;
+ALTER TABLE certificate_info ADD COLUMN aggchain_proof BLOB;
 ALTER TABLE certificate_info ADD COLUMN finalized_l1_info_tree_root VARCHAR;
-ALTER TABLE certificate_info_history ADD COLUMN aggchain_proof VARCHAR;
+ALTER TABLE certificate_info_history ADD COLUMN aggchain_proof BLOB;
```

Comment:
Shall these changes be part of a new migration file? It is the same doubt we had for bridge service: when deploying these changes, are they doing the fresh start, or just make an upgrade (use the newer binaries against the old data)?

Reply:
No. This is not an official release.